src/style.css

```diff
@@ -72,7 +85,7 @@ button {
 }
 
 label {
-    font-size: 20;
+    font-size: 16;
```

Comment:
I personally think 16 looks a bit small. Maybe 18 is a good compromise? I like that better personally